protocol/messenger_communities.go

```diff
@@ -982,7 +982,11 @@ func (m *Messenger) ShareCommunity(request *requests.ShareCommunity) (*Messenger
 		message := &common.Message{}
 		message.ChatId = pk.String()
 		message.CommunityID = request.CommunityID.String()
-		message.Text = fmt.Sprintf("Community %s has been shared with you", community.Name())
```

Comment:
Just a style thing: Could've left this line and remove the `else` case below

Reply:
fixed!